index.php

```diff
@@ -24,8 +26,5 @@
  * @brief Wrapper for OpenID Authentication Plugin.
  *
  */
-require_once('OpenIDPlugin.inc.php');
 
 return new OpenIDPlugin();
```

Comment:
You should be able to remove this wrapper script entirely, I think. If you try and get stuck on it, let me know.

Reply:
Removed the `index.php` file all together and everything seems to work fine